playbook-website/config/webpack/environment.js

```diff
@@ -22,6 +22,18 @@ environment.config.merge({
         include: /node_modules/,
         type: 'javascript/auto',
       },
+      {
+        test: /\.(js|mjs)$/,
+        exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader',
+          options: {
+            presets: [
+              '@babel/preset-env',
+            ],
+          },
+        },
+      },
```

Comment:
This is not the way to implement this loader. Webpacker already provides a `babel-loader` itself which you will need to get like this:

```js
const babelLoaderConfig = environment.loaders.get('babel')
const babelLoaderIndex = babelLoaderConfig.use.findIndex(config => { return config.loader === 'babel-loader' })
babelLoaderConfig.use[babelLoaderIndex].options.presets.unshift('@babel/preset-env')
```

Though I would caution and suggest first logging the output of `..options.presets` from the presented code snippet above first. This will ensure you are aware of any other presets being loaded from Webpacker's default configuration and help you to ensure you do not override those and deliver a breaking change.